apps/coscrad-frontend/src/configurable-front-matter/data/content.config.SAMPLE.ts

```diff
@@ -84,4 +84,8 @@ export const contentConfig: ConfigurableContent = {
     },
     notFoundMessage: 'No result found.',
     loadingMessage: 'Loading',
+    theme: {
```

Comment:
Do not use a custom schema for this object. Build the type from MUI's theme type.